discovery_engine_core/core/src/stack/filters.rs

```diff
@@ -116,4 +126,52 @@ mod tests {
             "Mensch mit d\u{00fc}sterer Prognose: \"Kollektiv versagt!\" N\u{00e4}chste Pandemie wird schlimmer als Covid-19",
         ]);
     }
+
+    #[test]
+    fn test_filter_title() {
+        let malformed_articles: Vec<Article> = serde_json::from_str(include_str!(
+            "../../test-fixtures/articles-invalid-title.json"
+        ))
+        .unwrap();
+        assert_eq!(2, malformed_articles.len());
```

Comment:
we usually call the assertion like `assert_eq!(actual, expected)`, hence i'd suggest to switch the arguments here and in the other tests (i know that we don't have a style guidline yet, still for better readability).
```suggestion
        assert_eq!(malformed_articles.len(), 2);
```